Mathmagician.Tests/FibonacciTest.cs

```diff
@@ -52,6 +52,49 @@ public void FibonacciEnsureListCount()
 
         }
 
+
+        public int fibotest(int a, int b)
+        {
+
+            int temp = a;
+            a = b;
+            b = temp + b;
+            return b;
+        }
```

Comment:
What's going on here is you are testing some functionality that isn't in your Fibonacci.cs anymore. I think this test is unnecessary, unless you move this method back into your Fibonacci.cs